dawn/renderer/actions/test/ConsoleActions.test.js

```diff
@@ -1,10 +1,32 @@
 import { expect } from 'chai';
-import { showConsole, hideConsole } from '../ConsoleActions';
+import { showConsole, hideConsole, updateConsole, clearConsole } from '../ConsoleActions';
 
-describe('console action creator', () => {
-  it('create showConsole action', () => {
-    expect(showConsole()).to.deep.equal({
+
+describe('console actions creator', () => {
+  // Needs to be using the same seed as the action creator.
+  it('should create an action to show console', () => {
+    const expectedAction = {
       type: 'SHOW_CONSOLE',
-    });
+    };
+    expect(showConsole()).to.deep.equal(expectedAction);
+  });
+  it('should create an action to hide console', () => {
+    const expectedAction = {
+      type: 'HIDE_CONSOLE',
+    };
+    expect(hideConsole()).to.deep.equal(expectedAction);
+  });
+  it('should create an action to update console', () => {
+    const expectedAction = {
+      type: 'UPDATE_CONSOLE',
+      consoleOutput: 0,
```

Comment:
Sorry this was not clear from the context, but `consoleOutput` should be a string in this case.